llmfoundry/models/llm_embed/gte_finetune_lm.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2024 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Any, Mapping
+
+import torch
+import torch.distributed as dist
+from transformers import AutoModel
+from transformers.modeling_outputs import CausalLMOutputWithPast
+
+from llmfoundry.models.llm_embed.modeling_llm_embed import ContrastiveModel
+
+
+class FinetuneEmbeddingModel(ContrastiveModel):
```

Comment:
the file is called `gte_finetune_lm` but the class is called `FinetuneEmbeddingModel`, can we make the filename more general and also take out the `lm` since it's not a language model?